modules/metering-install-operator.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-installing-metering.adoc
+
+[id="metering-install-operator_{context}"]
+= Installing the Metering Operator
+
+To install the Metering Operator first create a dedicated OpenShift project for
```

Comment:
Recommend not using "dedicated" here - it might get confused with our OpenShift dedicated product.